1.10/Dockerfile

```diff
@@ -8,7 +8,10 @@ FROM centos/s2i-core-centos7
 EXPOSE 8080
 EXPOSE 8443
 
-ENV NGINX_VERSION=1.10
+ENV NAME=nginx \
+    NGINX_VERSION=1.10 \
+    NGINX_SHORT_VER=110 \
+    VERSION=0 \
```

Comment:
@phracek the backslash here needs to be removed, build fails because of this.

Reply:
sorry for stupid typo. Fixed.